python/afdko/otfautohint/hinter.py

```diff
@@ -1848,14 +1900,20 @@ def calcInstanceStems(self, glidx):
 
                     iSS = iSSl[sidx][ul]
                     found = False
+                    assert self.glyph
                     if seg0.isBBox():
                         if seg0.isGBBox():
                             pbs = self.glyph.getBounds(None)
                         else:
+                            # I don't believe this can be reached, because
```

Comment:
This is probably just so obscure a corner case that none of our tests exercise it and no one has hit it yet, so we didn't catch the bug. I'll take a look at some point. 

`seg0.isBBox() and not seg0.isGBBox()` implies that the stems for seg0 were generated from the bounding box of the segment itself because nothing else worked. Only very unusual splines will have that problem.